webapp/tests/karma/ts/pipes/date.pipe.spec.ts

```diff
@@ -313,35 +318,56 @@ describe('date pipes rendering', () => {
       get: sinon.stub().resolvesArg(0),
     };
 
-    TestBed
-      .configureTestingModule({
-        imports: [
-          AsyncPipe,
-          AgePipe,
-          AutoreplyPipe,
-          DayMonthPipe,
-          FullDatePipe,
-          RelativeDayPipe,
-          RelativeDatePipe,
-          SimpleDateTimePipe,
-          SimpleDatePipe,
-          StatePipe,
-          WeeksPregnantPipe,
-          TestComponent,
-        ],
-        providers: [
-          { provide: RelativeDateService, useValue: relativeDate },
-          { provide: FormatDateService, useValue: formatDate },
-          { provide: TranslateService, useValue: translate },
-        ]
-      })
-      .compileComponents();
+    // Create a sanitizer instance
+    domSanitizer = new MockDomSanitizer();
+
+    TestBed.configureTestingModule({
+      imports: [
+        AsyncPipe,
+        AgePipe,
+        AutoreplyPipe,
+        DateOfDeathPipe,
+        DayMonthPipe,
+        FullDatePipe,
+        RelativeDayPipe,
+        RelativeDatePipe,
+        SimpleDateTimePipe,
+        SimpleDatePipe,
+        StatePipe,
+        WeeksPregnantPipe,
+        MaterialTestModule,
+        TestComponent,
+      ],
+      providers: [
+        { provide: RelativeDateService, useValue: relativeDate },
+        { provide: FormatDateService, useValue: formatDate },
+        { provide: TranslateService, useValue: translate },
+        { provide: DomSanitizer, useValue: domSanitizer },
+      ]
+    });
   });
 
   afterEach(() => {
     sinon.restore();
   });
 
+  // Helper function to safely get element text or undefined
+  const getElementText = (selector) => {
+    const element = fixture.nativeElement.querySelector(selector);
+    return element?.innerText;
+  };
+
+  // Helper function to safely get element attribute or undefined
+  const getElementAttribute = (selector, attribute) => {
+    const element = fixture.nativeElement.querySelector(selector);
+    return element?.getAttribute(attribute);
```

Comment:
I'm not sure this is going to be helpful. 
Suppose if this element doesn't exist, due to a change that someone makes or some bug, the test will start failing with `expected undefined to equal blabla` and no other errors. Then it will be quite hard to figure out what is going on. 
If we remove the nullchecks, then the test blows up and displays the error directly making debugging one step easier.